tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -334,8 +334,10 @@ def _run(self):
                     out_lines = stdout.split('\n')
                     connected = True
                     for index in range(4, len(out_lines) - 2):
-                        if out_lines[index].split('\t')[2].strip() != 'Connected':
-                            connected = connected and False
+                        node_status_det = out_lines[index].split('\t')
+                        if len(node_status_det) > 2:
+                            if node_status_det[2].strip() != 'Connected':
+                                connected = connected and False
```

Comment:
can this be connected=False?
I feel connected = connected and False is equivalent of connected=False....
Please correct me if I am missing something